example/Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM python:3.7
+FROM python:3.10.4-slim-buster
 
 ENV DEFAULT_KERNEL python3
 
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends git \
+ && apt-get purge -y --auto-remove \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
Apparently the official Python 3.10 Docker images don't come with git installed anymore.